src/normalize/pathoverlay.ts

```diff
@@ -130,7 +132,7 @@ export class PathOverlayNormalizer implements NonFacetUnitNormalizer<UnitSpecWit
     // FIXME: determine rules for applying selections.
 
     // Need to copy stack config to overlayed layer
-    const stackProps = stack(markDef, encoding);
+    const stackProps = stack(initMarkdef(markDef, encoding, config), encoding);
```

Comment:
This is technically a tech debt, since normalizer shouldn't call methods from an init phase. 

So I'd at least add

```ts
// FIXME: normalizer shouldn't call methods from an init phase. 
```